Automattic/wp-calypso
# Jetpack Cloud: Prevent restores and downloads on non-existent and ineligible activities

#### Changes proposed in this Pull Request

* Support new single activity endpoint
* Add Error screen when navigating to `/backups/:siteSlug/restore/:rewindId` or `/backups/:siteSlug/download/:rewindId` for an activity that is not rewindable
<img width="726" alt="Screen Shot 2020-05-01 at 4 02 47 PM" src="https://user-images.githubusercontent.com/2810519/80847698-779fc900-8bc5-11ea-9e2a-2e67ac9dd466.png">
* Add Error screen when navigating to `/backups/:siteSlug/restore/:rewindId` or `/backups/:siteSlug/download/:rewindId` for an activity that does not exist
<img width="711" alt="Screen Shot 2020-05-01 at 4 03 21 PM" src="https://user-images.githubusercontent.com/2810519/80847715-81c1c780-8bc5-11ea-8db4-c9509cfd7ee0.png">

#### Testing instructions

1. For a site that has real-time backup use the activity log to navigate to an activity that pre dates backups being activated.
1. Use the actions button to both "restore" and "download"
1. Verify the page matches the "is not rewindable" screenshot above
1. Add a digit to the rewind id in the url to make an invalid rewind id
1. Verify the page matches the "activity does not exist" screenshot above
